### DISCUSSION TITLE
VERSION ANNOUNCEMENT: 6.2

### DISCUSSION THREAD

**Discussion by DejanMilicic** [ravendb/ravendb#19306] [Announcements]
# VERSION ANNOUNCEMENT: 6.2

Today, we are releasing a new minor version, 6.2. This version is marked as LTS - **L**ong **T**erm **S**upport.  Such a version receives updates, fixes, and minor additions for an extended period - 12 months after the following LTS version is released. Considering our release cadence, the next LTS version will be released in 12-15 months, and 6.2 will receive support for an additional 12 months after that. This means you can count on no less than 24 months of support for version 6.2.

Here is what we are bringing to you:

## More integrations/interoperability: ETL to Azure Queue Storage

We expanded the set of supported ETL destinations with Azure Queue Storage. With a few lines of JS code, you can transform RavenDB documents and push them to AQS. From there, other components of your distributed system can pick up these messages and process them.

## RavenDB <3 DevOps `#1`:  OpenTelemetry Support

Are you already using OpenTelemetry to monitor various components of your system? Starting with v6.2, RavenDB joins the team. Your database leverages OpenTelemetry SDK to send metrics data via the OpenTelemetry Protocol, or you can use the RavenDB Prometheus endpoint. Find more details in the  #[documentation](https://ravendb.net/docs/article-page/6.2/csharp/server/administration/monitoring/open-telemetry)

## ~~RavenDB <3 DevOps `#2`:  Single file deployment~~

~~The server folder has been consolidated from over 500 files and folders to just five. This significant reduction simplifies the management of your RavenDB installation, making upgrades much more manageable by minimizing the number of files you need to replace with each new version.~~

## More control over sharding: Sharding by Prefix

[Prefixed Sharding](https://ravendb.net/docs/article-page/6.2/csharp/sharding/administration/sharding-by-prefix) gives you more control over the shard placement of your documents. The semantic structure of document IDs will direct RavenDB to place related documents together. This is useful in various scenarios where you need geo-distribution or data sovereignty and in simple scenarios where you want to minimize roundtrips by placing customers and their orders together in the same Shard.

## Small things that make life easier: Studio Omni Search

Ever had a problem finding your way around in Studio? A new search box is at the top of the Studio window, and it can help you. Press Ctrl+K, type in a term you are interested in, and you will get a list of Documents, Collections, Indexes, Tasks, Settings, and Studio Commands matching that term. As we add more and more features to Studio and organize them, they are inevitably getting buried down, sometimes even three or four levels deep. From now on, with just a few keystrokes, you can find what you are looking for quickly.

## Some Breaking changes

[Server Breaking Changes](https://ravendb.net/docs/article-page/6.2/csharp/migration/server/server-breaking-changes)

[Client Breaking Changes](https://ravendb.net/docs/article-page/6.2/csharp/migration/client-api/client-breaking-changes)

[Test Driver Breaking Changes](https://ravendb.net/docs/article-page/6.2/csharp/migration/embedded/testdriver-breaking-changes)

**Comment by ppekrol** [comment]
Due to the significantly reduced debugging capabilities when Single File Deployment is used, as of version 6.2.3 (released 21.01.2025), we have decided to remove that capability. At the moment, all tooling has issues reading memory dumps from those instances, and it is unknown when these will be resolved. We hope that one day we can revisit the idea, but for now, it is not possible.

If you are updating from pre-6.2, you will see absolutely no difference in the layout. Updates from 6.2.0 or 6.2.1 will not require any additional work, but you might notice significantly larger amount of files in the package.
